umibukela/templates/partners.html

```diff
@@ -43,24 +43,14 @@
             <th>Province</th>
           </tr>
         </thead>
+{% for partner in partners %}
         <tr>
-          <td>Tshedza Development Project</td>
+          <td><a href="{% url 'partner' partner.slug %}">{{ partner.short_name }}</a></td>
+          <td><a href="{% url 'site-result' site_name='folweni' result_id='4' %}"></a></td>
           <td></td>
-          <td>Social Development</td>
-          <td>Gauteng</td>
```

Comment:
I figured these were the sector and province of the site of the latest cycle results but I'll check with Jen. I'm therefore adding sector and provice fields to the Site model. I was thinking an enum for province and a Sector model with a many to one relationship from Site to Sector.